src/bin/test_userproc_even.c

```diff
@@ -0,0 +1,29 @@
+int main();
+int _start() {
+    return main();
+
+    // invoke system call that kills the process
+}
+
+#include "../syscall.h"
```

Comment:
Would it make some sense to have the makefile copy syscall.h into that directory? Or just have it live there entirely and have the nunya.iso make reference it one directory up? I think it would be more annoying to have the ../ in every user program we write than one weird line in the makefile